### DISCUSSION TITLE
Hexrd find-orintations/find-grains: "-c" vs "-f"

### DISCUSSION THREAD

**Discussion by kkp5295** [dmi-3D/dmi-3D.github.io#14] [Data processing]
I needed to run hexrd find-orientations and wanted to overwrite the existing files of the same name. Should I use the "-f" or "-c" extension and what is the difference? Is the same true for hexrd fit-grains?

**Comment by rachelelim** [comment]
Running _hexrd find-orientations -c_ will start at the beginning of the data processing and re-make the eta-omega maps. _hexrd find-orientations -f will us_e the previously calculated eta-omega maps. Using _hexrd fit-grains -f_ will force the program to overwrite any previously created grains.out file.
